app/handlers/users/start.py

```diff
@@ -9,8 +9,8 @@
 
 
 @dp.message_handler(commands=['start'])
-async def start_bot(msg: types.Message):
-    await on_start_cmd_notify(dp, msg.from_user)
+async def start_bot(msg: Message):
+    await on_start_cmd_notify(dp)
```

Comment:
Here bug. You give in arguments `dispatcher` but on_start_cmd_notify get `from_user`
```suggestion
    await on_start_cmd_notify(from_user)
```

Reply:
`AttributeError: 'Dispatcher' object has no attribute 'get_mention'`